test/schemaValidation.test.ts

```diff
@@ -2094,4 +2094,20 @@ obj:
     const result = await parseSetup(content);
     assert.equal(result.length, 0);
   });
+
+  it('value should match as per schema const on boolean', async () => {
+    schemaProvider.addSchema(SCHEMA_ID, {
+      type: 'object',
+      properties: {
+        prop: {
+          const: true,
+          type: 'boolean',
+        },
+      },
+    });
+    const content = `prop: false`;
```

Comment:
The failing case was that `prop: true` was flagged as incorrect. It would be nice to test both `prop: true` and `prop: false` in this test case.